src/storage/prisma.ts

```diff
@@ -309,7 +309,7 @@ export class PrismaStorage implements IDriveStorage {
                     header
                 );
             },
-            { isolationLevel: 'Serializable' }
+            { isolationLevel: 'Serializable', maxWait: 10000, timeout: 20000 }
```

Comment:
Any operations that took more than 5 seconds to process where being discarded. Increased the timeout to 20s.
This should no longer be needed when we implement optimistic updates